### PR TITLE
Create 20170912-prediction-of-huawei-sales-of-vidhance-models.md

### PR DESCRIPTION
**Pull request by vvizard-**
Proposed article about Huawei sales predictions

### CODE REVIEW

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+date: 2017-09-12T15:20:00+02:00
+tags: [sales]
+title: Prediction of Huawei sales of Vidhance models
+author: Mr IMINT
+---
+
+# Prediction of Huawei sales of Vidhance models
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: You don't have to give the title as an header. It is added automatically. This also means that you can reduce header levels by one in the rest of the document.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Disclaimer
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: I suggest calling it _Background_ instead. It sounds a little less boring.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+| **P8** | 2015 April | - | - | - | - | - | - | - | - | - | - | _4.77M_ | - |
+| **P9** | 2016 April | - | **>2.6M** [(1)][P9_2MONTH] | - | - | **6M** [(1)][P9_5MONTH] | - |- |  **9M** [(1)][P9_9MONTH] | - | - | **12M** [(1)][P9_12MONTH] | - |
+| **P10** | 2017 Mar | - | - | **6M** [(1)][P10_3MONTH] | - | - | - | - | - | - | - | - | - |
+| **Mate 8** | 2015 Dec | **>1M** [(1)][M8_1MONTH] | - | - | _3.68M_ | - | - | - | - | - | - | - | 7M [(1)][M8_12MONTH] |
+| **Mate 9** | 2016 Dec | - | - | - | **5M** [(1)][M9_4MONTH] | - | - | **8.5M** [(1)][M9_4MONTH] | - | - | - | - | - |
+| **Honor 8** | 2016 Jul | - | **1.5M** [(1)][H8_2MONTH] | - | - | - | - | - | - | - | - | - | - |
+| **Honor 9** | 2017 Jun | **>1M** [(1)][H9_1MONTH] | - | - | - | - | - | - | - | - | - | - | - |
+
+### Mate 9 Family
+The Mate 9 family includes the "Mate 9", "Mate 9 Pro" and "Mate 9 Porsche design". The "Mate 9 Lite" is not included, as it does not run Vidhance software. The Mate 9 Family was announced in October 2016, and went on sale in China in November. Availability in most of western Europa, along with the US was not in place until January 2017.
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Maybe state very early in the article that Lite are totally different models and then define the Mate 9 Series and use that everywhere.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+The only data we got for the P10 so far, is that it had sold above 6 million units in it's 3rd month. The P9 didn't reach 6 million until it's 5th month. So the P10 seem to have outsold the P9 by a __huge margin__ for it's first quarter. Unfortunately that's all the data we got for the P10 so far, but I expect to get an update on this at end of 2017 (or before), as it certainly looks on track to shatter the previous sales-record from the company.
+
+___Huaweis previous flagship record is held by the P9 which sold 12 million units within it's first 11 months. The only datapoint we got for P10 so far, shows it's selling 36% better than the P9 did at that point in time. As Huawei have seen a overall 20.6% increase in 2017H1 on total unit shipments (all models), and P10 seemingly a very big success with the consumers, I'd say a _modest_ 25% increase over P9 could be realistic. That will put P10 sales after 11 months on market somewhwere around 15-16 million units. Competition will be harder this autumn than the last, because of the anniversary iPhone, and a Samsung Note 8 that so far haven't exploded.___
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Second last word should be "hasn't".

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+### Honor 9
+This is tough. Detailed sales about any Honor model are hard to come by. We know that yearly sales of the Honor subsidiary accounted for approximately 60 million units of Huawei's total 139.3 shipped units.  Other than that, the only data available is that Honor 8 sold 1.5 million units by it's second month after release, while the Honor 9 (with Vidhance) have exceeded 1 million during it's first month on market. These numbers seem to be in line with the _Huawei Mate 8_ and the _Huawei P8 Lite_. Which respectively sold 7 million units at 12 months for the Mate, and 10 million units at 9 months for the P8. That's quite a span.
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: "while the Honor 9 (with Vidhance) _has_ exceeed"

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+
+__Based on strong year-on-year growth for the Honor brand, and the "Honor 9" realtively cheap chinese price, compared to the P10, one could expect this to sell a lot of units. But it got a lot of competition from other Huawei / Honor models. Earlier this year, Honor released the "Honor V9" in china, which was rebranded into the "Honor 8 Pro" in international markets. It's specifications and price is pretty equal to both the Honor 9, and the Huawei P10. So quite a lot of potential Honor 9 customers, probably end up with a "Honor V9 / Honor 8 Pro" or a P10 instead. As we know P10 sales look incredible strong so far, my guess is that the Honor 9 will sell somewhere around 8-10 million units in it's first 12 months. But I'm feeling less confident in this prediction than for the Mate 9 and P10.__
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Try only to highlight a few words at a time. These long paragraphs are not suitable to skim through. Instead try to find the few words in every part that actually highlights the conclusion.

**Review by simonmika — COMMENTED** [review]
Good in general. Maybe it might be worth splitting it into one piece talking about which models contain Vidhance and another about the potential volumes. The second can reference the first. Just a suggestion. Great work by the way.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+We also know that the Mate 9 had sold 8.5 million units within it's 7th month. This seems to put it right "up there" with P9 sales, following quite similar sales volumes as the P9 through it's 4th to 7th months beeing available. The Mate-series probably doesn't usually sell as well as the P-series, so this could seem a bit strange, but when adding in the fact that 2017H1 have seen a 20.6% total volume increase for Huawei smartphones, a new Mate-series selling comparable to last years P-series might not be as unlikely anymore. 
+
+Looking at all data we got for all relevant Huawei/Honor phones, monthly unit shipping seems to be very strong (actually near it's peak) around the 7th month after a release. It should be safe to assume Mate 9 is still selling in relative good volumes. 
+
+___Based on similarity with P9 in sales volumes so far, the fact that Mate-series is usually outsold by the P-series, but accounting for that 2017 so far have shown a 20.6% unit shipment increase from Huawei, I'm guessing the Mate 9 will sell somwhere in the ballpark of the P9 in it's first 12 months. More specifically 10-12 million units.___
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Use single underscore.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md (hunk 4 above)
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Single underscore again.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+
+__Based on strong year-on-year growth for the Honor brand, and the "Honor 9" realtively cheap chinese price, compared to the P10, one could expect this to sell a lot of units. But it got a lot of competition from other Huawei / Honor models. Earlier this year, Honor released the "Honor V9" in china, which was rebranded into the "Honor 8 Pro" in international markets. It's specifications and price is pretty equal to both the Honor 9, and the Huawei P10. So quite a lot of potential Honor 9 customers, probably end up with a "Honor V9 / Honor 8 Pro" or a P10 instead. As we know P10 sales look incredible strong so far, my guess is that the Honor 9 will sell somewhere around 8-10 million units in it's first 12 months. But I'm feeling less confident in this prediction than for the Mate 9 and P10.__
+
+## Conclusion
+The current models from Huawei / Honor with Vidhance included is
+* Mate 9 family
+* P10 family
+* Honor 9
+
+> __I expect these three models to combined sell between 33 to 38 million units during their first 12 months__.
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Don't use underscore for the full quote. And make the quote as self explanatory as possible while still being short enough for Twitter.

Comment (simonmika) [reply content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: And move it higher up in the document (just a few lines) and leave a version of it here as well.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+## The bigger picture for Huawei smartphones
+
+![P8 Lite shipment trend](http://gulfnews.com/polopoly_fs/1.1979574.1487244664!/image/576257842.jpg "P8 Lite shipment trend")
+
+ * 2014: 75 million units shipped [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2015: 108 million units shipped [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2016: 139.3 million units shipped (+30.2% year-on-year)  [(source: Huawei)](http://consumer.huawei.com/en/press/news/2017/hw-u-035103/)
+ * 2017H1: 73.01 million units shipped (+20.6% year-on-year) [(source: Huawei)](http://www.huawei.com/za/news/za/huawei-consumer-business-group-announces-2017-h1-business-results)
+ 
+> _"By the third quarter of 2016, 60% of Huawei’s smartphone shipments were mid- and high-priced devices, a striking reversal of its past reliance on cheap handsets."_
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: Don't add quote signs when using `>` as they will be added by the css. Also remember that quotes a flown outside of the regular text so you might too have a shorter version in the quote and a longer version left here. Also don't emphasize full sentences.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+* Imint AB have never confirmed Vidhance beeing part of the P-series P10 model, but according to firmware analysis, [I'm personally sure it does](https://twitter.com/vvizard_/status/846701491704475648).
+* Imint have never confirmed Vidhance beeing part of an Honor-series phone. But [I'm personally sure](https://twitter.com/vvizard_/status/872532160258899973) Vidhance is included in one of their 2017 flagship phones, the _"Honor 9"_.
+
+My claims regarding Vidhance beeing part of the P10 have been peer reviewed by others, and my claim of Vidhance beeing used in the Honor 9 is a peer review of another persons original claim. There are several community members who have run analysis on these firmwares, and come to the same conclusions. For the remainder of this article, Vidhance beeing used in Mate 9, P10 and Honor 9 is considered a fact by the author.
+
+Both the Mate 9 and the P10 comes in several variants. The variants are
+|Name| Running Vidhance stabilization?|
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: You need another newline before the table for it too work.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+* Imint AB have never confirmed Vidhance beeing part of the P-series P10 model, but according to firmware analysis, [I'm personally sure it does](https://twitter.com/vvizard_/status/846701491704475648).
+* Imint have never confirmed Vidhance beeing part of an Honor-series phone. But [I'm personally sure](https://twitter.com/vvizard_/status/872532160258899973) Vidhance is included in one of their 2017 flagship phones, the _"Honor 9"_.
+
+My claims regarding Vidhance beeing part of the P10 have been peer reviewed by others, and my claim of Vidhance beeing used in the Honor 9 is a peer review of another persons original claim. There are several community members who have run analysis on these firmwares, and come to the same conclusions. For the remainder of this article, Vidhance beeing used in Mate 9, P10 and Honor 9 is considered a fact by the author.
+
+Both the Mate 9 and the P10 comes in several variants. The variants are
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: End line with `:`.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+![P8 Lite shipment trend](http://img.c.huaweistatic.com/content/dam/huawei-cbg-site/en/mkt/press/news/2016/31609-1.jpg "P8 Lite shipment trend")
+
+### P9
+Although the P8 Lite managed to sell above 10 million units, P9 was credited by Huawei as beeing the first _"flagship"_ device to do so.
+
+### General sales
+Here is a table trying to give every known sales volumes by offical (or at least credible) sources, for all Huawei and Honor models that might be relevant for predicting the sales volumes of the models running Vidhance. _Italic data points_ are estimations based upon other __known datapoints, marked in bold__. Paranthesis mark the source of information for the datapoint.
+
+| Model | Released | Month 1 | Month 2 | Month 3  | Month 4  | Month 5  | Month 6  | Month 7  | Month 8  | Month 9  | Month 10  | Month 11  | Month 12|
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: This table is too wide for the page. I might find a way to fix it but we should consider rearranging the data somehow.

File: content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+The current models from Huawei / Honor with Vidhance included is
+* Mate 9 family
+* P10 family
+* Honor 9
+
+> __I expect these three models to combined sell between 33 to 38 million units during their first 12 months__.
+
+That is counting each model from it's release, and 12 months out. They are likely to continue selling some millions (combined) even after 12 months, but a lack of good datapoints at this make it hard to predict from after 12 months.
+
+---
```
Comment (simonmika) [inline content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md]: This line makes no sense as the links below actually will show up at their usage.

**Review by simonmika — CHANGES_REQUESTED** [review]
I will have to update the css to handle the images.

**Comment by vvizard-** [comment]
Regarding CSS and images. What was the problem? Is it something you will update, or should I make changes? If so, what?

**Comment by simonmika** [comment]
I already fixed the images.

**Review by simonmika — APPROVED** [review]

**Pull request merged**